src/fragments/lib-v1/storage/android/configureaccess/50_customization.mdx

```diff
@@ -87,7 +87,18 @@ Amplify.addPlugin<Plugin<*>>(
 </Block>
 </BlockSwitcher>
 
-Add the IAM policy that corresponds with the prefixes defined above to enable read, write and delete operation for all the objects under path *myPublicPrefix/*:
+### Defining IAM Policy for Custom Prefix
+
+You will need to update the IAM policy to enable read, write and delete operation for all the objects under your custom prefix path.
+
+Directly updating the relevant IAM role policies in AWS policies will be overwritten for future calls of Amplify push.
```

Comment:
Is this also true for that if a developer creates a new set of IAM policy to define the custom prefix permissions?